guides/common/modules/proc_creating-a-webhook-template.adoc

```diff
@@ -0,0 +1,19 @@
+[id="creating-a-webhook-template_{context}"]
+= Creating a Webhook Template
+
+Use the following procedure to create a webhook template in the {ProjectWebUI}.
+
+.Procedure
+
+. In the {ProjectWebUI}, navigate to *Administer* > *Webhooks Templates*.
+. Click *Clone an existing template* or *Create Template*.
+. Enter a name for the template.
+. Use the editor to make changes to the template payload.
++
+A webhook HTTP payload must be created using {Project} template syntax.
++
+For more information, see {ManagingHostsDocURL}appe-red_hat_satellite-managing_hosts-template_writing_reference[Template Writing Reference] in _Managing Hosts_ and for available template macros and methods, visit `/templates_doc` on {ProjectServer}.
```

Comment:
The anchor has been changed, see `guides/doc-Managing_Hosts/topics/Template_Writing.adoc:4:[[appe-Managing_Hosts-Template_Writing_Reference]]`

```suggestion
For more information, see {ManagingHostsDocURL}appe-Managing_Hosts-Template_Writing_Reference[Template Writing Reference] in _Managing Hosts_ and for available template macros and methods, visit `/templates_doc` on {ProjectServer}.
```